ci/pipeline.yml

```diff
@@ -0,0 +1,74 @@
+---
+resource_types:
+- name: bosh2-deployment
+  type: docker-image
+  source:
+    repository: cloudfoundry/bosh-deployment-resource
+    tag: v2.8.1
+
+resources:
+- name: git
+  type: git
+  source:
+    uri: git@github.com:govau/dta-frontend.git
+    private_key: {{git_private_key}}
+    branch: master
+    ignore_paths:
+    - releases/*
+- name: git-push
+  type: git
+  source:
+    uri: git@github.com:govau/dta-frontend.git
+    private_key: {{git_private_key}}
+    branch: master
+- name: github-release
+  type: github-release
+  source:
+    owner: govau
+    repository: dta-frontend
+    access_token: {{git_personal_access_token}}
+- name: version
+  type: semver
+  source:
+    driver: git
+    uri: git@github.com:govau/boshreleases.git
+    branch: master
+    file: dta-frontend.version
+    initial_version: 0.3.0
+    private_key: {{git_boshreleases_private_key}}
+
+jobs:
+- name: build-it
+  plan:
+  - aggregate:
+    - get: version
+      params:
+        bump: patch
```

Comment:
Is it also possile to have a different build job for bumping the major/minor version numbers as well?

Reply:
good question.  i think it depends on how much we want to adopt semantic versioning.  many BOSH releases don't bother.